js/util/is_char_in_unicode_block.js

```diff
@@ -0,0 +1,268 @@
+    // 'Currency Symbols': (char) => char >= 0x20A0 && char <= 0x20CF,
+    // 'Combining Diacritical Marks for Symbols': (char) => char >= 0x20D0 && char <= 0x20FF,
+    // 'Letterlike Symbols': (char) => char >= 0x2100 && char <= 0x214F,
+    // 'Number Forms': (char) => char >= 0x2150 && char <= 0x218F,
+    // 'Arrows': (char) => char >= 0x2190 && char <= 0x21FF,
+    // 'Mathematical Operators': (char) => char >= 0x2200 && char <= 0x22FF,
+    // 'Miscellaneous Technical': (char) => char >= 0x2300 && char <= 0x23FF,
+    // 'Control Pictures': (char) => char >= 0x2400 && char <= 0x243F,
+    // 'Optical Character Recognition': (char) => char >= 0x2440 && char <= 0x245F,
+    // 'Enclosed Alphanumerics': (char) => char >= 0x2460 && char <= 0x24FF,
+    // 'Box Drawing': (char) => char >= 0x2500 && char <= 0x257F,
+    // 'Block Elements': (char) => char >= 0x2580 && char <= 0x259F,
+    // 'Geometric Shapes': (char) => char >= 0x25A0 && char <= 0x25FF,
+    // 'Miscellaneous Symbols': (char) => char >= 0x2600 && char <= 0x26FF,
+    // 'Dingbats': (char) => char >= 0x2700 && char <= 0x27BF,
+    // 'Miscellaneous Mathematical Symbols-A': (char) => char >= 0x27C0 && char <= 0x27EF,
+    // 'Supplemental Arrows-A': (char) => char >= 0x27F0 && char <= 0x27FF,
+    // 'Braille Patterns': (char) => char >= 0x2800 && char <= 0x28FF,
+    // 'Supplemental Arrows-B': (char) => char >= 0x2900 && char <= 0x297F,
+    // 'Miscellaneous Mathematical Symbols-B': (char) => char >= 0x2980 && char <= 0x29FF,
+    // 'Supplemental Mathematical Operators': (char) => char >= 0x2A00 && char <= 0x2AFF,
+    // 'Miscellaneous Symbols and Arrows': (char) => char >= 0x2B00 && char <= 0x2BFF,
+    // 'Glagolitic': (char) => char >= 0x2C00 && char <= 0x2C5F,
+    // 'Latin Extended-C': (char) => char >= 0x2C60 && char <= 0x2C7F,
+    // 'Coptic': (char) => char >= 0x2C80 && char <= 0x2CFF,
+    // 'Georgian Supplement': (char) => char >= 0x2D00 && char <= 0x2D2F,
+    // 'Tifinagh': (char) => char >= 0x2D30 && char <= 0x2D7F,
+    // 'Ethiopic Extended': (char) => char >= 0x2D80 && char <= 0x2DDF,
+    // 'Cyrillic Extended-A': (char) => char >= 0x2DE0 && char <= 0x2DFF,
+    // 'Supplemental Punctuation': (char) => char >= 0x2E00 && char <= 0x2E7F,
+    'CJK Radicals Supplement': (char) => char >= 0x2E80 && char <= 0x2EFF,
+    'Kangxi Radicals': (char) => char >= 0x2F00 && char <= 0x2FDF,
+    'Ideographic Description Characters': (char) => char >= 0x2FF0 && char <= 0x2FFF,
+    'CJK Symbols and Punctuation': (char) => char >= 0x3000 && char <= 0x303F,
+    'Hiragana': (char) => char >= 0x3040 && char <= 0x309F,
+    'Katakana': (char) => char >= 0x30A0 && char <= 0x30FF,
+    // 'Bopomofo': (char) => char >= 0x3100 && char <= 0x312F,
+    'Hangul Compatibility Jamo': (char) => char >= 0x3130 && char <= 0x318F,
+    // 'Kanbun': (char) => char >= 0x3190 && char <= 0x319F,
+    // 'Bopomofo Extended': (char) => char >= 0x31A0 && char <= 0x31BF,
```

Comment:
This block doesn’t appear in OSM, but it could appear in custom data. It should be vertical and get ideographic breaking.